### PR TITLE
[Ryujinx.SDL2.Common] Address dotnet-format issues

### PR DESCRIPTION
**Pull request by TSRBerry**
Base PR: #4670

### CODE REVIEW

File: src/Ryujinx.SDL2.Common/SDL2Driver.cs [hunk 1]
```diff
@@ -80,8 +77,15 @@ public void Initialize()
                 }
 
                 // First ensure that we only enable joystick events (for connected/disconnected).
-                SDL_GameControllerEventState(SDL_DISABLE);
-                SDL_JoystickEventState(SDL_ENABLE);
+                if (SDL_GameControllerEventState(SDL_IGNORE) != SDL_IGNORE)
```
Comment (AcK77) [inline src/Ryujinx.SDL2.Common/SDL2Driver.cs:80]: This replace `SDL_GameControllerEventState(SDL_DISABLE);` so why it's now using `SDL_IGNORE` ?

**Review by marysaka — APPROVED** [review]
same comment as Ack otherwise lgtm

File: src/Ryujinx.SDL2.Common/SDL2Driver.cs (hunk 1 above)
Comment (TSRBerry) [reply src/Ryujinx.SDL2.Common/SDL2Driver.cs:80]: I have no idea. I'll change it!

Comment (TSRBerry) [reply src/Ryujinx.SDL2.Common/SDL2Driver.cs:80]: Nvm, I figured it out.
https://wiki.libsdl.org/SDL2/SDL_GameControllerEventState
The only valid values for this function are: `SDL_IGNORE`, `SDL_ENABLE` and `SDL_QUERY`

**Review by AcK77 — APPROVED** [review]

**Pull request merged**